teos/data_dir/teos.toml

```diff
@@ -0,0 +1,33 @@
+# API
+api_bind = "127.0.0.1"
+api_port = 9814
+tor_control_port = 9051
+onion_hidden_service_port = 9814
+tor_support = false
+
+# RPC
+rpc_bind = "127.0.0.1"
+rpc_port = 8814
+
+# bitcoind
+btc_network = "regtest"
+btc_rpc_user = "exch"
+btc_rpc_password = "goodpass"
```

Comment:
not really

Reply:
lol, do not include your `datadir` in the PR

Reply:
removed